enterprise/backend/src/metabase_enterprise/sso/integrations/jwt.clj

```diff
@@ -75,7 +76,7 @@
                                                      (group-names->ids group-names)
                                                      (all-mapped-group-ids))))))
 
-(defn- login-jwt-user
+(defn- get-jwt-data
```

Comment:
usually Clojure avoids the `get-` prefix. I was going to say it should be called `jwt-data`, but it's more than that (it returns a map with a key called `:jwt-data`, so…)

Reply:
I changed it to `session-data` which I think is a more accurate way to describe that function (at least more than just `jwt-data`)